### PR TITLE
added basic search functionality

### PR DESCRIPTION
**Pull request by feiss**

### CODE REVIEW

File: site/index.html [hunk 1]
```diff
@@ -21,17 +21,15 @@
...
-        <div class="filtertype">
+        <!-- <div class="filtertype">
           <img src="site/img/filter.png" alt="filter by type">
           <a href="#">Filter by type</a>
-        </div>
+        </div> -->
         <div class="search">
           <img src="site/img/search.png" alt="search">
-          <input type="text" placeholder="Search...">
+          <input type="text" placeholder="Search..." onkeyup="search(this.value)">
```
Comment (donmccurdy) [inline site/index.html]: perhaps `oninput` here, to support copy/paste.

File: site/index.html [hunk 2]
```diff
@@ -85,6 +83,16 @@ <h1 class="component__title">
       ga('send', 'pageview');
 
       new window.Blazy();
+
+      var componentList = document.querySelectorAll('.component');
+      function search(s) {
+        var text = ('innerText' in document.body)? 'innerText' : 'textContent';
+        for (var i = 0; i < componentList.length; i++) {
+          componentList[i].style.display = (componentList[i][text].indexOf(s) !== -1) ? 'flex' : 'none';
```
Comment (donmccurdy) [inline site/index.html]: would say `... ? null : 'none'`, as `null` will remove inline styles and revert to whatever is in the CSS.

File: site/index.html (hunk 1 above)
Comment (feiss) [inline site/index.html]: Cool, I didn't know that event :D

File: site/index.html (hunk 2 above)
Comment (feiss) [inline site/index.html]: yup, sure!

I'll add a "Filtering by "xxx" (yy results):" text below the header when searching, too.

**Comment by feiss** [comment]
I've just added search summary and a search clear button.

captura de pantalla 2017-06-20 a las 17 50 49

@donmccurdy set the `oninput` event. `null` didn't work, I left it like it was before.

**Comment by fernandojsg** [comment]
Looking great thanks!

**Pull request merged**